coral/src/app/main.tsx

```diff
@@ -45,13 +47,17 @@ function prepare(): Promise<void | ServiceWorkerRegistration> {
 
 prepare().then(() => {
   root.render(
-    <React.StrictMode>
-      <QueryClientProvider client={queryClient}>
-        <AquariumContext>
-          <RouterProvider router={router} />
-          {DEV_MODE && <ReactQueryDevtools />}
-        </AquariumContext>
-      </QueryClientProvider>
-    </React.StrictMode>
+    // <React.StrictMode>
```

Comment:
Is it required we disable `StrictMode`?

Reply:
No, that was for debugging / development. The branch is not cleaned up yet, there are probably also some console logs and stuff still :D